worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -111,65 +149,276 @@ public Job process(final String jobUuid) {
     /**
      * Process in individual contract
      *
-     * @param contract - the contract to process
+     * @param contracts - all the contract to process
      * @param job - the job in which the contract belongs
      * @param month - the month to search for beneficiaries for
      * @param outputDirPath - the location of the job output
      * @param progressTracker - the progress tracker which indicates how far the job is along
      * @throws ExecutionException when there is an issue with searching
      * @throws InterruptedException - when the search is interrupted
      */
-    void processContract(Contract contract, Job job, int month, Path outputDirPath, ProgressTracker progressTracker) throws ExecutionException, InterruptedException {
-        log.info("Job [{}] - contract [{}] ", job.getJobUuid(), contract.getContractNumber());
-        // Retrieve the contract beneficiaries
+    private void processContracts(List<Contract> contracts, Job job, int month, Path outputDirPath,
```

Comment:
This method is way too big to understand plainly and would be significantly helped by several refactoring to methods for the inner operations.  Then, this method would read more plainly and have the same level of abstraction.